src/test/java/kg/jarkyn/cobspecserver/WorkerTest.java

```diff
@@ -0,0 +1,23 @@
+package kg.jarkyn.cobspecserver;
+
+import kg.jarkyn.cobspecserver.doubles.ClientDouble;
+import kg.jarkyn.cobspecserver.doubles.ResponseControllerDouble;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class WorkerTest {
```

Comment:
What should happen when exceptions on the socket communication occur? At the moment, `RequestParser` and `StreamHandler` just rethrow them...
Can they occur?